man/cdn.md

```diff
@@ -39,8 +44,19 @@
 由 FiveCDN 提供，境内使用 火山引擎 + 白山云 CDN ，境外使用 Cloudflare CDN。
 支持所有 Github 官方仓库的所有 Tags / Branches 作为 :version 的值。
 
-如果您在使用本 CDN 的过程中遇到了问题，请联系 Mail:[im#aehxy.com](mailto:im@aehxy.com)。
+如果您在使用本 CDN 的过程中遇到了问题，请联系 Mail:[im@aehxy.com](mailto:im@aehxy.com)。
+
+### bscdn.tbedu.top
+
+地址: `https://bscdn/:version/public`
+
+![](https://blessing-skin-manual.vercel.app/api/cdn?cdn=bs-cdn.yecdn.com/:version/public)
+![](https://blessing-skin-manual.vercel.app/api/cdn?cdn=bs-cdn.yecdn.com/:version/public&version=4.4.0)
```

Comment:
```suggestion
![](https://blessing-skin-manual.vercel.app/api/cdn?cdn=bscdn.tbedu.top/:version/public&version=4.4.0)
```

Reply:
Okay, I will listen to these suggestions to modify them